src/rules/tsUtils.ts

```diff
@@ -2,10 +2,34 @@
 import { basename } from 'path';
 import { ESLintUtils } from '@typescript-eslint/experimental-utils';
 import { version } from '../../package.json';
+import {
+  AST_NODE_TYPES,
+  TSESTree,
+} from '@typescript-eslint/experimental-utils';
 
 const REPO_URL = 'https://github.com/jest-community/eslint-plugin-jest';
 
 export const createRule = ESLintUtils.RuleCreator(name => {
   const ruleName = basename(name, '.ts');
   return `${REPO_URL}/blob/v${version}/docs/rules/${ruleName}.md`;
 });
+
+enum DescribeAlias {
```

Comment:
@SimenB Should I replace this enum with a type.

Reply:
nah, an enum is fine 🙂